src/base/bittorrent/peerinfo.h

```diff
@@ -38,17 +38,9 @@ class QBitArray;
 
 namespace BitTorrent
 {
+    struct PeerAddress;
```

Comment:
we have no such rule but I would place struct declarations after class declarations.